Result viewer (#259)
viktormalik/diffkemp: README.md

The DiffKemp binary is then located in `bin/diffkemp`.

> For running result viewer it is necessary to have installed:

Reviewer: ```suggestion
For running the result viewer, it is necessary to have installed:
```

> * Node.js (14)

Reviewer: Based on the CI, we support Node.js 14, 16, 18, and 19, right? Maybe `>= 14.x` would be better here?

* npm (8)

Manually building with `cmake` will automatically install necessary packages for

> the result viewer and build optimized version of the result viewer.

Reviewer: ```suggestion
the result viewer and build an optimized version of the result viewer.
```

  automatically. The `--stdout` option causes the diffs to be printed to
  standard output.

> Additionally you can run **result viewer** to get visualisation of found
> differences.

Reviewer: ```suggestion
Additionally, you can run **result viewer** to get a visualisation of the found
differences.
```

> - **Result viewer** takes directory with output of compare phase. 

Reviewer: ```suggestion
- **Result viewer** takes the directory with the output of the compare phase. 
```

  It is invoked via:
  ```
  diffkemp view COMPARE_OUTPUT_DIR
  ```

>   It prepares necassary files and runs static server. You can open result viewer
>   in your browser on shown address and port.

Reviewer: ```suggestion
  It prepares the necessary files and runs a static server. The command displays the URL that you can use to access the result viewer.
```

The [rhel-kernel-get.py](https://github.com/viktormalik/rhel-kernel-get) script can also be used
to download and configure the aforementioned kernels.

The result viewer contains unit tests and integration tests
which can be run by:

>     cd view && npm test -- --watchAll

Reviewer: Add blank line before the next section.
```suggestion
    cd view && npm test -- --watchAll
    
```
